quill-spark/src/test/scala/io/getquill/context/spark/GithubExample.scala

```diff
@@ -0,0 +1,90 @@
+package io.getquill.context.spark
+
+import java.io.File
+import java.net.URL
+
+import scala.language.postfixOps
+import scala.sys.process._
+
+import org.apache.spark.sql.SparkSession
+
+import io.getquill.QuillSparkContext.Ord
+import io.getquill.QuillSparkContext._
+
+case class User(
+  id:          String,
+  login:       String,
+  gravatar_id: String,
+  url:         String,
+  avatar_url:  String
+)
+
+case class Repo(
+  id:   String,
+  name: String,
+  url:  String
+)
+
+case class Activity(
+  id:         String,
+  `type`:     String,
+  actor:      User,
+  repo:       Repo,
+  created_at: String,
+  org:        User
+)
+
+object GithubExample extends App {
```

Comment:
I think that we should create SPARK.md which would show all advantages of quill against raw spark. 
And add link to this file in readme, since imo, existing example in readme does not show "Tired of the #scala #spark untyped madness?" :)

Reply:
Good point. I'm actually planning to write a blog post based on this example. I think I'll wait for a release with the new module first, though.